tests/RobotFramework/tests/cumulocity/shell/c8y_Command

```diff
@@ -1,4 +1,4 @@
 [exec]
 topic = "c8y/s/ds"
 on_message = "511"
-command = "/etc/tedge/operations/command_handler.sh"
\ No newline at end of file
+command = "/etc/tedge/operations/command"
```

Comment:
I'm just providing context, why this line is needed.

There was a subtle change in the `robotframework-devicelibrary` on the `Transfer To Device` keyword which changed the way the `command-handler.sh` file is copied to the device. Now the keyword supports renaming files whilst coping if the target folder does not have a trailing slash `/`.